tools/aws_cli.py

```diff
@@ -213,6 +213,7 @@ def UploadToBucket(args: 'argparse.Namespace') -> None:
 
   logger.info('File successfully uploaded.')
 
+
```

Comment:
remove extra blank line

Reply:
This should bring the function in line with the rest of the file, with 2 blank lines between each high level `def`